lib/amazonka/src/Amazonka/Auth/Background.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
Not sure if you copied over the original bang patterns or not - but if so, the original reasoning behind them seems to be lost. I'd consider just removing them where they don't clearly beat you about the head with their obviousness, or alternatively mark this entire module `{-# LANGUAGE Strict #-}`.

Reply:
OK.